src/components/ColorPicker.tsx

```diff
@@ -144,8 +145,10 @@ const Picker = ({
               (event.currentTarget as HTMLButtonElement).focus();
               onChange(_color);
             }}
-            title={`${_color} — ${keyBindings[i].toUpperCase()}`}
-            aria-label={_color}
+            title={`${t(`colors.${_color.replace("#", "")}`)}${
+              !isTransparent(_color) ? ` (${_color})` : ""
+            } — ${keyBindings[i].toUpperCase()}`}
```

Comment:
I think this is sort of impacting readability so we can try to simplify it
`t(colors.${_color.replace("#", "")}` can be assigned to a variable `ariaLabel` and reused here and line 151 as well.

Reply:
Done in https://github.com/excalidraw/excalidraw/pull/3871/commits/05f27abe062fcb42285f8ce049ea5e8fe8602965.

Reply:
It's used on `title` and `aria-label`, so went for a generic name.